views/register.hbs

```diff
@@ -29,6 +29,18 @@
             <input id="email" name="email" type="text" data-auto-trim placeholder="{{{__ "enter email"}}}" value="{{formValues.email}}" size="30" class="pure-input-1 login-form-input">
         </div>
 
+        {{#if signupTeamsTransfer}}
+        <div class="pure-control-group">
+          <h4>Teams suggested by invite link, check box to join</h4>
```

Comment:
May need to add explanation or link to explanation of what teams are.

Via Eloquence: text will need to be internationalized before merge

Reply:
If you pass the full team objects along, you can access the internationalized team names via `{{{mlString this.name}}}``` inside the `{{#each}}` loop.